assets/src/edit-story/app/media/local/reducer.js

```diff
@@ -29,7 +29,7 @@ const INITIAL_STATE = {
   ...COMMON_INITIAL_STATE,
   processing: [],
   processed: [],
-  mediaType: types.LOCAL_MEDIA_TYPE_ALL,
+  mediaType: '',
```

Comment:
@barklund I've reverted this (unbreaking the unit tests), LMK if this was required for the media pane dropdown -- didn't seem so but perhaps I missed something.